organization/_posts/2025/2025-03-27-coordination.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "HSF Coordination Meeting #285, 27 March 2025"
+layout: plain_toc
+---
+
+## Attending
+
+Present/Contributing: Graeme Stewart, Eduardo Rodrigues, Claire Antel, Stefan Roiser, Ruslan Mashinistov, Ines Ochoa, Tommaso Lari, Torre Wenaus, Matthew Feickert, Alexander Moreno, Juan Miguel Carceller, Michel Jouvin, Luke Kreczko, Liz Sexton-Kennedy, Christian Wessel, Sapta Battacharya, Richa Sharma 
+
+Apologies/Contributing: Mark Neubauer (US/EU Daylight shift caused meeting conflict for me)
+
+
+## News, general matters, announcements
+
+### European Strategy Update (EPPSU) final stages
+
+We are in the final stages of the process. We have endorsments from ALICE, ATLAS, Belle II, ePIC, LHCb and MCnet.
+
+There were a few ~minor comments (adding references, mainly) as endorsements arrived. The continually updated final draft is in [CERNBox](https://cernbox.cern.ch/s/XQ3wEp3L8UOVjjb).
+
+CMS and DUNE are expected to endorse this week and WLCG will endorse when all LHC experiments are on board.
+
+CMS asked how they could cite the document, so we have decided to put it onto Zenodo now, to have an immediate DOI. We will also put this final version on arXiv (but the arXiv publication delay means that that DOI will come too late).
+
+### HSF/WLCG Workshop in May
+
+- Abstracts have been coming our way - close to 20 now.
+- Sessions are in preparation, in good shape but not yet published.
+- Attendance seems low cf. previous workshops.
+    - Please sign up!
+    - A reminder will be sent to the HSF Forum.
+
+### HSF Seminar Series and Compute Accelerator Forum
+
+[Seminar Indico Category](https://indico.cern.ch/category/18810/).
+[Compute & Accelerator Forum Category](https://indico.cern.ch/category/12741/)
+
+The HSF Seminar on *technical debt in software* happened yesterday: <https://indico.cern.ch/event/1520114/>.
+- Around 30(?) people in attendance (~6 people in the room)
+- YouTube recording already uploaded: <https://youtu.be/BkSiusiqhSk?si=mTqOtJZa2IOv5PS0>.
+- No slides uploaded yet but speaker will do so soon - not many plots but plots shown very interesting, e.g.:
+<img src="https://codimd.web.cern.ch/uploads/upload_c89f24ba057a1a8882f01ed6843e2e99.png" alt="drawing" width="450" height="300"/>
```

Comment:
Unsure how to have this changed for the site. Else we simply remove the "e.g. ..."?

Reply:
I think we need to take the PNG out of CodiMD and put it on our own website, because I can't think that URL will be so stable...

Reply:
No  prob. I can put it in the same directory as the minutes, organization/_posts/2025, unless you see some other better place?

Reply:
Trying exactly the above ...

Reply:
I can't see what is wrong but the snapshot does not show up in the netlify preview :(.

Reply:
The way that Jekyll processes the site means that images in that `organization` source directory are ignored. I have moved the image into `images/coordination` and it looks fine now.